modules/imgproc/perf/perf_warp.cpp

```diff
@@ -7,12 +7,14 @@ namespace opencv_test {
 
 enum{HALF_SIZE=0, UPSIDE_DOWN, REFLECTION_X, REFLECTION_BOTH};
 
+CV_ENUM(Mat_Type, CV_8UC3, CV_8UC4, CV_16UC3, CV_32FC3)
+
```

Comment:
no need in extra line here.

Reply:
Will remove later. Need this for testing purposes.